doc/plugins.md

```diff
@@ -169,6 +169,10 @@ The list of plugins:
     — rewrite element with rehype
 *   [`rehype-sanitize`](https://github.com/rehypejs/rehype-sanitize)
     — sanitize HTML
+*   [`rehype-semantic-images`](https://github.com/benjamincharity/rehype-semantic-images)
+    — Enrich HTML images with semantic elements and customizable features
```

Comment:
Could this be added to the list in alphabetical order?
Between `rehype-sectionize` and `rehype-shift-heading`?

Reply:
Whoops.. letters are hard! 

I've fixed that and better aligned my casing.

I've also released a new version of `rehype-semantic-plugins` addressing your optional comments. Thanks again!!